lib/Elastica/Aggregation/Filters.php

```diff
@@ -59,6 +59,30 @@ public function addFilter(AbstractQuery $filter, $name = null)
     }
 
     /**
+     * @param bool $other_bucket
+     *
+     * @return $this
+     */
+    public function setOtherBucket($other_bucket)
+    {
+        if (!is_bool($other_bucket)) {
```

Comment:
We use lowerUpper case for the variable names across Elastica. Could you change it to $otherBucket? Same for the other variable names in this PR